### PR TITLE
Npc sheet redesign

### PR DESCRIPTION
**Pull request by guswadsworth12**
Updated styling to various elements.

### CODE REVIEW

File: templates/actor/actor-npc-sheet.hbs [hunk 1]
```diff
@@ -1,59 +1,108 @@
...
+          </div>
+      </div>
+     </div> 
 
-    {{!-- Notes Tab --}}
-    <div class="tab notes" data-group="primary" data-tab="notes">
-      {{> "systems/essence20/templates/actor/parts/actor-notes.hbs"}}
+     <div class="npc-abilities-2">
+      <div class="npc-hangup">
+          <span class="resource flex-group-center"><h2>{{localize 'E20.hangUps'}}</h2></span>
```
Comment (phildominguez) [inline templates/actor/actor-npc-sheet.hbs:80]: Making this an `h1` will make the boxes line up. Or maybe make them all `h2`s to shrink them a bit?

**Comment by phildominguez** [comment]
Noticing some files that have a mix of 2 and 4 space indentations. The codebase is generally 2-space (inherited from Boilerplate) so you may have to change your VSCode settings:
image
I have Detect Indentation turned off as well, but I'm not sure if that matters. You can auto-format with Shift + Alt + F.

File: templates/actor/parts/actor-npc-defenses.hbs [hunk 2]
```diff
@@ -1,9 +1,14 @@
-<div class="defenses-section">
```
Comment (phildominguez) [inline templates/actor/parts/actor-npc-defenses.hbs:1]: Any objections to making this look the same as the movement section? I'm also thinking we don't really need the `Defenses` header on it as well. Thoughts?

File: templates/actor/parts/actor-traits.hbs [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<h3>{{localize 'E20.traits'}}</h3>
+<span class="resource flex-group-center"><h1>{{localize 'E20.traits'}}</h1></span>
```
Comment (phildominguez) [inline templates/actor/parts/actor-traits.hbs:1]: Been meaning to standardize the header text in en.json so they're all capitalized. I'm gonna add a ticket to do a pass on those.

File: system.json [hunk 4]
```diff
@@ -22,7 +22,7 @@
   "primaryTokenAttribute": "health",
   "secondaryTokenAttribute": "power",
   "url": "https://github.com/WookieeMatt/Essence20",
-  "manifest": "https://raw.githubusercontent.com/WookieeMatt/Essence20/beta/system.json",
-  "download": "https://github.com/WookieeMatt/Essence20/releases/download/beta/Essence20-beta.zip",
+  "manifest": "https://github.com/WookieeMatt/Essence20/releases/download/beta/system.json",
```
Comment (phildominguez) [inline system.json:25]: Not the raw file anymore?

**Review by phildominguez — CHANGES_REQUESTED** [review]
Looking really good my dude

File: templates/actor/actor-npc-sheet.hbs (hunk 1 above)
Comment (guswadsworth12) [reply templates/actor/actor-npc-sheet.hbs:80]: changed to h1

File: templates/actor/parts/actor-npc-defenses.hbs (hunk 2 above)
Comment (guswadsworth12) [reply templates/actor/parts/actor-npc-defenses.hbs:1]: image

Updated to reflect.

File: system.json (hunk 4 above)
Comment (guswadsworth12) [reply system.json:25]: This is a temporary change because i want to keep 1 beta private release. Which does not allow for us to pull them the way want.

**Review by phildominguez — APPROVED** [review]
💯

**Pull request merged**